bin/ci

```diff
@@ -19,6 +19,12 @@ def main(options)
     wait_until(unencrypted_vms_st)
   end
 
+  github_runner_test_cases = options[:test_cases].select { _1.include?("github_runner") }
```

Comment:
I think we need to have a way to trigger github runner cases separately, so we need to pass that information to e2e tests somehow. Given that we expect tests will be running automatically, development experience overweigh the complexity.

Reply:
Within my tests preparing host takes ~12-15 mins. Since I've changed tests to run in parallel, github runner tests add ~12 mins (downloading images takes almost all) on top of it. I'm planning to keep E2E test runs periodic as we have right now. We can also trigger if we have changes related to github runner. Agree to moving larger host with faster network.

Reply:
I agree with the idea of parallelization, but we need to be cautious. To my knowledge, our E2E host has a capacity of 4 VMs (though I could be mistaken). The VmGroup test provisions 3 virtual machines. If we run two VmGroup tests in parallel, each might allocate 2 VMs and then wait for a third, potentially leading to a deadlock.

Reply:
As runner tests are starting after downloading image, it doesn't coincidence with vm tests. So, it is ok for now. We should also use bigger host for tests. I'll work on that in parallel.

Reply:
We have 2 vm group tests, they coincidence with each other 😄